locale/en.json

```diff
@@ -39,6 +39,7 @@
     "copy": "Copy"
   },
   "sidebar": {
+    "nodeFilter": "Node filter",
```

Comment:
every filter there is a node filter, so that's a very generic term. it does filter by the node's name, so...